### DISCUSSION TITLE
Cannot import mixture sampler experimentalist

### DISCUSSION THREAD

**Discussion by musslick** [AutoResearch/autora#515] [Using AutoRA] [answered]
When running 
`!pip install -U autora["experimentalist-sampler-mixture"]` in Google Colab, it says:

``WARNING: autora 3.0.1 does not provide the extra 'experimentalist-sampler-mixture'`` despite the mixture sampler being listed in autora's [pyproject.tml](https://github.com/AutoResearch/autora/blob/main/pyproject.toml). 

I'm wondering why the mixture sampler can't be installed?

@younesStrittmatter @hollandjg @blinodelka

**Comment by benwandrew** [comment]
i think there may be some discrepancy between `mixture-experimentalist` and `experimentalist-sampler-mixture`. we have included it in the parent as the latter but the release on PyPI says to [pip install via the former](https://pypi.org/project/mixture-experimentalist/#:~:text=pip%20install%20mixture%2Dexperimentalist).

**Reply by benwandrew** [reply]
no, maybe not... the pyproject.toml seems to [capture](https://github.com/AutoResearch/autora/blob/6fecbe22524e7772fad8f022f6f1ad6c6b6f312d/pyproject.toml#L77) the correct name of the dependency

**Reply by benwandrew** [reply]
either way, even if not the cause of the error, i think we should conform the naming here

**Comment by benwandrew** [comment, marked as answer]
ah, 3.0.1 does not in fact include the mixture experimentalist. we added if after that release. so need to do a new release of the parent for it to be included.

**Comment by younesStrittmatter** [comment]
Hi, I think the newest version does not have yet been released

**Comment by hollandjg** [comment]
Do we just need to make a new release - 3.1.0?

**Reply by hollandjg** [reply]
I don't see any reason not to, do let me know if you need a hand with it!

**Reply by benwandrew** [reply]
done!
